### PR TITLE
Add X509_VERIFY_PARAM_set1_ip support

### PR DESCRIPTION
**Pull request by miyazakh**

### CODE REVIEW

File: src/ssl.c [hunk 1]
```diff
@@ -25186,6 +25186,134 @@ int wolfSSL_X509_VERIFY_PARAM_set1_ip_asc(WOLFSSL_X509_VERIFY_PARAM *param,
...
+        *   to re-construct IP address in ascii.
+        */
+        buf = (char*)XMALLOC(max_ipv6_len, NULL, DYNAMIC_TYPE_TMP_BUFFER);
+
+        if (!buf) {
+            WOLFSSL_MSG("failed malloc");
+            return ret;
+        }
+        p = buf;
+        for (i = 0;i < 16;i += 2) {
```
Comment (cconlon) [inline src/ssl.c:25260]: Please add spaces after semicolons, thanks

File: src/ssl.c [hunk 2]
```diff
@@ -25186,6 +25186,134 @@ int wolfSSL_X509_VERIFY_PARAM_set1_ip_asc(WOLFSSL_X509_VERIFY_PARAM *param,
...
+               write_zero = 1;
+           }
+           else {
+               if (i != 0)
+                *p++ = ':';
+               XSPRINTF(p, "%x", val);
+           }
+           /* sanity check */
+           if (XSTRLEN(buf) > max_ipv6_len) {
+               printf("The target ip adress exceeds buffer length(40)\n");
```
Comment (cconlon) [inline src/ssl.c:25277]: Please use WOLFSSL_MSG instead of printf

File: src/ssl.c [hunk 3]
```diff
@@ -25186,6 +25186,134 @@ int wolfSSL_X509_VERIFY_PARAM_set1_ip_asc(WOLFSSL_X509_VERIFY_PARAM *param,
...
+             * Therefore, adding : and null termination
+             */ 
+                 p++;
+                 *p++ = ':';
+                *p = '\0';
+            }
+        }
+    }
+    else {
+        WOLFSSL_MSG("iplen is zero, do nothig");
```
Comment (cconlon) [inline src/ssl.c:25300]: Spelling - "nothig" should be "nothing"

**Review by cconlon — CHANGES_REQUESTED** [review]

File: src/ssl.c (hunk 1 above)
Comment (miyazakh) [reply src/ssl.c:25260]: Add spaces after the smicolons.
Thanks!

File: src/ssl.c (hunk 2 above)
Comment (miyazakh) [reply src/ssl.c:25277]: Use WOLFSSL_MSG instead of printf

File: src/ssl.c (hunk 3 above)
Comment (miyazakh) [reply src/ssl.c:25300]: Fix typo. Thanks

**Review by cconlon — APPROVED** [review]

**Pull request merged**